defaults/main.yml

```diff
@@ -486,6 +486,11 @@ apt__apt_preferences__dependent_list:
     reason:  'Issues in wheezy package - http://debian.distrosfaqs.org/debian-user/wheezy-irqbalance/'
     by_role: 'debops.apt'
 
+  - package: 'irqbalance'
+    backports: [ 'jessie' ]
```

Comment:
You can just request that in the same entry as the Wheezy `irqbalance` preference:
```
- package: 'irqbalance'
  backports: [ 'wheezy', 'jessie' ]
```
It will be configured on both of these releases.